packages/extension/src/config.ts

```diff
@@ -77,6 +77,11 @@ export const EmbedChainInfos: ChainInfo[] = [
     ],
     coinType: 118,
     features: [],
+    gasPriceStep: {
+      low: 0.008,
```

Comment:
Is there any limitation on the precision of these values. If we are aiming at a 5 nano FET minimum then this value would be `0.000000005` I guess. Do we run into any issues with the underlying `number` type in javascript in this case?

Reply:
Yes! I think I misunderstood the gas price / fee spreadsheet a little initially.

Reply:
It looks like we did run into a limitation after all:
```
Uncaught Error: invalid decimal: 4e-9
    at new Dec (popup.bundle.js:351498)
    at FeeConfig.getFeeTypePrimitive (popup.bundle.js:339793)
    at FeeConfig.<anonymous> (popup.bundle.js:339709)
...
```